src/cmd/property.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/opslevel/opslevel-go/v2023"
+
+	"github.com/opslevel/cli/common"
+
+	"github.com/spf13/cobra"
+)
+
+var createPropertyDefinitonCmd = &cobra.Command{
+	Use:   "property-definition",
+	Short: "Create a property-definition",
+	Long:  `Create a property-definition`,
+	Example: fmt.Sprintf(`
+cat << EOF | opslevel create property-definition -f -
+%s
+EOF`, getExamplePropertyDefinitionYaml()),
+	Run: func(cmd *cobra.Command, args []string) {
+		input, err := readResourceInput[opslevel.PropertyDefinitionInput]()
+		cobra.CheckErr(err)
+		newPropertyDefinition, err := getClientGQL().CreatePropertyDefinition(*input)
+		cobra.CheckErr(err)
+
+		fmt.Println(newPropertyDefinition.Id)
+	},
+}
+
+var getPropertyDefinition = &cobra.Command{
+	Use:        "property-definition",
+	Short:      "Get details about a property definition",
+	Long:       `Get details about a property definition`,
+	Args:       cobra.ExactArgs(1),
+	ArgAliases: []string{"ID"},
+	Run: func(cmd *cobra.Command, args []string) {
+		identifier := args[0]
+		result, err := getClientGQL().GetPropertyDefinition(identifier)
+		cobra.CheckErr(err)
+
+		common.WasFound(result.Id == "", identifier)
```

Comment:
`GetPropertyDefinition` will return an error anyways if the resource isn't found, so this line is unnecessary. This is the case with a LOT of code in tools that calls `GetSomeResource()`.

Reply:
Good call 👍